File: client/src/components/portals/Appointment.js
import React from "react";

function Appointment({ appt, handleOnClick }) {
	return (
		<tr onClick={() => handleOnClick(appt)}>
			<td>{appt.time}</td>
			<td>{appt.patient}</td>
			<td>{appt.doctor}</td>
		</tr>
	)
}

export default Appointment;

File: app/models/appointment.rb
class Appointment < ApplicationRecord
    belongs_to :doctor
    belongs_to :patient

		validates :patient_id, presence: true
		validates :doctor_id, presence: true
		validates :date, presence: true
		validates :time, presence: true
		# validates :date, presence: true, on_or_after: {Date.current}
end
